src/platforms/common/session-replay/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Set Up Session Replay
```

Comment:
Since this content will be exclusively for JavaScript, I wonder if the best location is under:

`src/platforms/javascript/common` instead of `src/platforms/common`. The latter causes inheritance to all platforms

Reply:
Moved. 

Native is on the roadmap, but it'll be sooo different that i don't think it'll be a problem (and we could move it back if we wanted)